controllers/changetierrequest/changetierrequest_controller_test.go

```diff
@@ -74,12 +74,12 @@ func TestChangeTierSuccess(t *testing.T) {
 		require.NoError(t, err)
 		murtest.AssertThatMasterUserRecord(t, "johny", cl).
 			HasTier(*teamTier).
-			AllUserAccountsHaveTier(*teamTier).
+			UserAccountHasNoTier(test.MemberClusterName).
```

Comment:
No, I wanted to keep the `UserAccountHasNoTier(test.MemberClusterName).` beause the MUR has two UserAccounts:
* in `test.MemberClusterName`
* in "another-cluster"
So I thought that it would be fine to verify it for both of them.

Reply:
Oh I see now. Added it back. It was a small change 🙂  dc7db4f160282d80104a819b10732c90fabd557a